---
title: "DIA-NN processing workflow"
output: html_document
date: "2022-09-05"
---

# DIA-NN software specification
For the testing purposes, version [1.8.1](https://github.com/vdemichev/DiaNN/releases/tag/1.8.1) of the DIA-NN software was installed and used.

# Test dataset
For the testing purposes, raw data from Fig.3 in [Bruderer et al. (2017)](https://pubmed.ncbi.nlm.nih.gov/29070702/) were used. 

The samples are **triplicates** of **two mixed proteome samples** (as described in Supplementary Table 2):

* Sample1: 1 (human) : 1 (c. elegans) : 1 (s. cerevisiae) : 1.3 (e. coli)
* Sample2: 1 (human) : 1.1 (c. elegans) : 1.2 (s. cerevisiae) : 1 (e. coli)

# DIA-NN workflow

We fill perform the library-free analysis in two steps: firstly creating the in silico library from provided FASTA databases, and then using our raw data with this library.

1. **Spectral library preparation**

In our case, working with 4 different organisms, we will need 4 different FASTA files, which will be used for spectral library generation.

Additionally, we will use a contaminant database (based on cRAP).

**Specific settings**:

* Protein inference is based on Isoform IDs, not Gene names
* Digestion is performed using Trypsin/P
* Precursor ion generation: check both FASTA digest for library-free search and Deep learning-based spectra.
* Directory for main output adjusted
* No MBR for this case
* Adjust the experiment name, save the pipeline







